confChamber.py

```diff
@@ -47,17 +48,19 @@
                       help="VThreshold1 DAC bump value for all VFATs", default=0)
     parser.add_argument("--zeroChan", action="store_true", dest="zeroChan",
                       help="Zero all channel registers")
+    parser.add_argument("--gemType",type=str,help="String that defines the GEM variant, available from the list: {0}".format(gemVariants.keys()),default="ge11")
+    parser.add_argument("--detType",type=str,help="Detector type within gemType. If gemType is 'ge11' then this should be from list {0}; if gemType is 'ge21' then this should be from list {1}; and if type is 'me0' then this should be from the list {2}".format(gemVariants['ge11'],gemVariants['ge21'],gemVariants['me0']),default=None)
```

Comment:
Defaulting this to `None` should raise an `OHTypeException` based on these lines:

https://github.com/cms-gem-daq-project/cmsgemos/blob/48e2f269dac0947be0101d2a9f0d943752e2ce61/gempython/tools/vfat_user_functions_xhal.py#L21-L22

When you use this at line 61 I expect an error to be thrown.